ext/liquid_c/vm.h

```diff
@@ -3,6 +3,7 @@
 
 #include <ruby.h>
 #include "block.h"
+#include "resource_limits.h"
```

Comment:
It doesn't look like this is needed for this header, so this include could be moved into `vm.c`